src/guides/v2.3/graphql/interfaces/downloadable-product.md

```diff
@@ -14,12 +14,12 @@ The `DownloadableProduct` object contains the following attributes:
 
 Attribute | Type | Description
 --- | --- | ---
-`downloadable_product_links` | [`DownloadableProductLinks`] | An array containing information about the links for this downloadable product
-`downloadable_product_samples` | [`DownloadableProductSamples`] | An array containing information about samples of this downloadable product
+`downloadable_product_links` | [[DownloadableProductLinks]](#downloadableProductLinks) | An array containing information about the links for this downloadable product
+`downloadable_product_samples` | [[DownloadableProductSamples]](#downloadableProductSamples)  | An array containing information about samples of this downloadable product
 `links_purchased_separately` | Int | A value of 1 indicates that each link in the array must be purchased separately
 `links_title` | String | The heading above the list of downloadable products
 
-### DownloadableProductSamples object
+### DownloadableProductSamples object {#downloadableProductSamples}
```

Comment:
Capitalize the "D" and update all references to the anchor.

Reply:
You missed this one. Also, you introduced an error in line 18 by linking to the wrong location.